File: Back-end/DotNetCore/TMS/Models/Tasks.cs
using System;
using System.Collections.Generic;

namespace TMS.Models;

public partial class Tasks
{
    public long Id { get; set; }

    public string? Description { get; set; }

    public DateTime? DueDate { get; set; }

    public int? Status { get; set; }

    public int StoryPoints { get; set; }

    public string? Username { get; set; }
}

File: Back-end/SpringBoot/src/main/java/com/TaskManagement/TM/model/Tasks.java
package com.TaskManagement.TM.model;

import com.TaskManagement.TM.Enum.TaskStatus;
import lombok.Getter;

import javax.persistence.*;
import java.time.LocalDate;

@Getter
@Entity
public class Tasks {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private int storyPoints;
    private LocalDate dueDate;

    private String username;
    private String description;

    private TaskStatus status;

    public void setId(Long id) {
        this.id = id;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public void setStoryPoints(int storyPoints) {
        this.storyPoints = storyPoints;
    }

    public void setDueDate(LocalDate dueDate) {
        this.dueDate = dueDate;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setStatus(TaskStatus status) { this.status = status; }
}
